addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -2132,3 +2132,43 @@ QUnit.test(
         assert.verifySteps(["Second RPC"]);
     }
 );
+
+QUnit.test("Newly created chat should be at the top of the direct message list", async () => {
+    const pyEnv = await startServer();
+    const [ userId1, userId2 ] = pyEnv["res.users"].create([
+        { name: "Jerry Golay" },
+        { name: "Albert"}
+    ]);
+    const [partnerId1] = pyEnv["res.partner"].create([
+        {
+            name: "Albert",
+            user_ids: [ userId2 ],
+        },
+        {
+            name: "Jerry Golay",
+            user_ids: [ userId1 ],
+        }
+    ]);
+    pyEnv["discuss.channel"].create({
+        channel_member_ids: [
+            Command.create({
+                is_pinned: true,
+                last_interest_dt: "2021-01-01 10:00:00",
+                partner_id: pyEnv.currentPartnerId,
+            }),
+            Command.create({ partner_id: partnerId1 }),
+        ],
+        channel_type: "chat",
+    });
+
+    const { openDiscuss } = await start();
+    await openDiscuss();
```

Comment:
No empty whitespace in test, it gives wrong impression at a glance on file that there are 2 tests here where it's a single test. Empty newlines add nothing except unnecessary confusion, better keep LOCs of test cohesive by having no arbitrary empty newlines